build.rs

```diff
@@ -0,0 +1,22 @@
+use std::env;
+
+const LIBCAPNG_LIB_NAME: &str = "cap-ng";
+const LIBCAPNG_LIB_PATH: &str = "LIBCAPNG_LIB_PATH";
+const LIBCAPNG_LINK_STATIC: &str = "LIBCAPNG_LINK_STATIC";
```

Comment:
For consistency with `libseccomp-sys`, could you please rename this to `LIBSECCOMP_LINK_TYPE` and mimic its behavior?